middleware/faq.rst

```diff
@@ -535,6 +535,251 @@ The resulting query logs can be useful for developers and database administrator
+                       deepCoadd_measMatchFull     216        0              0           78      0   294      294
+                              measure_metadata     216        0              0           78      0   294      294
+                                   measure_log     216        0              0           78      0   294      294
+                                 deepCoadd_ref      36        0              0           13      0    49       49
+                    mergeMeasurements_metadata      36        0              0           13      0    49       49
+                         mergeMeasurements_log      36        0              0           13      0    49       49
+                          deepCoadd_forced_src     216        0              0           78      0   294      294
+                      forcedPhotCoadd_metadata     216        0              0           78      0   294      294
+                           forcedPhotCoadd_log     216        0              0           78      0   294      294
+                                 deepCoadd_obj      36        0              0           13      0    49       49
+                     writeObjectTable_metadata      36        0              0           13      0    49       49
+                          writeObjectTable_log      36        0              0           13      0    49       49
+                                   objectTable      36        0              0           13      0    49       49
+                 transformObjectTable_metadata      36        0              0           13      0    49       49
+                      transformObjectTable_log      36        0              0           13      0    49       49
+                             objectTable_tract       0        0              0            1      0     1        1
+               consolidateObjectTable_metadata       0        0              0            1      0     1        1
+                    consolidateObjectTable_log       0        0              0            1      0     1        1
+     match_ref_truth_summary_objectTable_tract       0        0              0            1      0     1        1
+  match_target_truth_summary_objectTable_tract       0        0              0            1      0     1        1
+                   matchObjectToTruth_metadata       0        0              0            1      0     1        1
+                        matchObjectToTruth_log       0        0              0            1      0     1        1
+       matched_truth_summary_objectTable_tract       0        0              0            1      0     1        1
+  diff_matched_truth_summary_objectTable_tract       0        0              0            1      0     1        1
+                 compareObjectToTruth_metadata       0        0              0            1      0     1        1
+                      compareObjectToTruth_log       0        0              0            1      0     1        1
```

Comment:
This code block should also be indented by 4 characters.

Reply:
For the purposes of this quick FAQ, do we need to show the entire table here? Perhaps only showing 3-5 lines, and then ... ellipses to indicate that more lines are below, would be sufficient?

Reply:
PS - it looks like some of the headers for this table also became misaligned with the main table body as well.